Feature/inscription

Rhich opened the pull request.

Rhich committed:
inscription

Rhich committed:
inscription

Rhich committed:
inscription

franckPicardat committed:
Merge branches 'dev' and 'feature/inscription' of https://github.com/Master-DL/projet-ivvq-2017-2018-frodel into feature/inscription

Rhich committed:
test utilisateur valide

Rhich committed:
test utilisateur valide

Rhich committed:
test utilisateur valide

franckPicardat committed:
Merge branch 'feature/inscription' of https://github.com/Master-DL/projet-ivvq-2017-2018-frodel into feature/inscription

franckPicardat committed:
Merge branch 'feature/creer-un-voyage' of https://github.com/Master-DL/projet-ivvq-2017-2018-frodel into feature/inscription

franckPicardat committed:
tests Users

franckPicardat committed:
tests Users

franckPicardat committed:
tests Users

Rhich committed:
Créer un utilisateur #13
Mettre en place userService et UserRepository

Rhich committed:
Créer un utilisateur #13
ajout du test d'integration pour UserService

franckPicardat committed:
Merge branch 'feature/inscription' of https://github.com/Master-DL/projet-ivvq-2017-2018-frodel into feature/inscription

Rhich committed:
Créer un utilisateur #13
fixer les erreurs de merge

Rhich committed:
Créer un utilisateur #13
test service

franckPicardat committed:
Merge branch 'feature/inscription' of https://github.com/Master-DL/projet-ivvq-2017-2018-frodel into feature/inscription

Rhich committed:
Merge branch 'dev' into feature/inscription

Marianna-dl committed:
Merge branch 'dev' into feature/inscription

Marianna-dl commented:
Comme l'indique le rapport codecov, il y a des lignes dans UserService.java qui ne sont pas couvertes. Je vous laisse regarder si elles sont vraiment utiles/testées :-)
(Vous devriez arriver à plus de 70% de couverture je pense)

franckPicardat committed:
Merge branch 'feature/inscription' of https://github.com/Master-DL/projet-ivvq-2017-2018-frodel into feature/inscription

franckPicardat committed:
Créer un utilisateur #13
Modification de la classe UserService pour enlever les getter/setter non utiles

Rhich committed:
Créer un utilisateur #13
ajout du controleur utilisateur + test

Rhich committed:
Merge branch 'dev' into feature/inscription
